# GazeData.msg
float64 eye_angle
float64 head_orientation
float64 noise_level
